<ui version="4.0" >
 <class>WdgBumpmap</class>
 <widget class="QWidget" name="WdgBumpmap" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>336</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lblLayer" >
     <property name="text" >
      <string>&amp;Bumpmap layer:</string>
     </property>
     <property name="buddy" >
      <cstring>txtSourceLayer</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="txtSourceLayer" >
     <property name="toolTip" >
      <string>Bumpmapping is a process where two layers are combined to give one layer the illusion of depth. One layer will contain your image, the other a grayscale or black-and-white representation of height, which is the bumpmap. If you do not specify a bumpmap layer, the current layer will be used.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="bnRefresh" >
     <property name="text" >
      <string>&amp;Preview</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="grpType" >
     <property name="title" >
      <string>&amp;Type</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="radioLinear" >
        <property name="text" >
         <string>&amp;Linear</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSpherical" >
        <property name="text" >
         <string>&amp;Spherical</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSinusoidal" >
        <property name="text" >
         <string>S&amp;inusoidal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QGroupBox" name="grpOptions" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="chkCompensate" >
        <property name="text" >
         <string>&amp;Compensate for darkening</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkInvert" >
        <property name="text" >
         <string>I&amp;nvert bumpmap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkTiled" >
        <property name="text" >
         <string>&amp;Tile bumpmap</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QGroupBox" name="grpSettings" >
     <property name="title" >
      <string>Settings</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading</set>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel3" >
        <property name="text" >
         <string>&amp;Azimuth:</string>
        </property>
        <property name="buddy" >
         <cstring>dblAzimuth</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="dblAzimuth" >
        <property name="maximum" >
         <double>360.000000000000000</double>
        </property>
        <property name="value" >
         <double>135.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="textLabel6" >
        <property name="text" >
         <string>&amp;X offset:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QSpinBox" name="intXOffset" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel4" >
        <property name="text" >
         <string>&amp;Elevation:</string>
        </property>
        <property name="buddy" >
         <cstring>dblElevation</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="dblElevation" >
        <property name="minimum" >
         <double>0.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>90.000000000000000</double>
        </property>
        <property name="value" >
         <double>45.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="textLabel7" >
        <property name="text" >
         <string>&amp;Y offset:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QSpinBox" name="intYOffset" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel5" >
        <property name="text" >
         <string>&amp;Depth:</string>
        </property>
        <property name="buddy" >
         <cstring>dblDepth</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="dblDepth" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>65.000000000000000</double>
        </property>
        <property name="value" >
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="textLabel8" >
        <property name="text" >
         <string>&amp;Water level:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QSpinBox" name="intWaterLevel" />
      </item>
      <item row="3" column="2" >
       <widget class="QLabel" name="textLabel9" >
        <property name="text" >
         <string>&amp;Ambient light:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QSpinBox" name="intAmbient" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>txtSourceLayer</tabstop>
  <tabstop>radioLinear</tabstop>
  <tabstop>chkCompensate</tabstop>
  <tabstop>chkInvert</tabstop>
  <tabstop>chkTiled</tabstop>
  <tabstop>dblAzimuth</tabstop>
  <tabstop>dblElevation</tabstop>
  <tabstop>dblDepth</tabstop>
  <tabstop>bnRefresh</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
